uktrade/data-hub-frontend
# Reactify Large Capital Profile location section

## Description of change

This is the second stage of the work started in #5480

The location section of the Large Capital Profile page has now been reactified. The `EditLargeCapitalInvestorLocation` form has also been refactored and moved to the `modules` directory to live alongside the rest of the new Large Capital Profile code.

## Test instructions

Open the large capital profile tab for a company and open the location section. You should be able to view and edit the location data.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
